src/utils/sessions.js

```diff
@@ -6,31 +6,40 @@ import { CapacitorHttp } from '@capacitor/core';
 // TODO: will be different host from a mobile device, probably if (!window)
 let authUrl = `${window.location.origin}/api/login`;
 
-if (Capacitor.isNativePlatform()){
-  authUrl = "http://192.168.1.247:8080/api/login";
+if (Capacitor.isNativePlatform()) {
+  authUrl = "http://10.0.2.2:8080/api/login";
```

Comment:
We should probably create global constants like `NATIVE_BASE_URL` or whatever, but I've hardcoded this for now.